scripts/vmbuild.rb

```diff
@@ -172,7 +175,11 @@ def verify_run(output)
         $log.info `scp #{destination} #{FILE_SERVER_ACCOUNT}@#{FILE_SERVER}:#{file_rdu_dir.join(file_name)}`
       end
     end
+
+    # The final image is moved out of STORAGE_DIR at this point, delete all other files created during build
+    temp_file_uuid.each { |uuid| FileUtils.rm(Dir.glob("#{STORAGE_DIR}/#{uuid}.*"), :verbose => true) }
```

Comment:
It works with `FileUtils.rm` - probably because everything is ran and owned by root.